Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -31,8 +36,13 @@ class PayPalDemoViewController: FeatureBaseViewController {
         NSLayoutConstraint.activate([
             payPalButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
             payPalButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.layoutSpacing),
-            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
-            payPalButton.heightAnchor.constraint(equalToConstant: Constants.textFieldHeight)
+            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: 50),
```

Comment:
```suggestion
            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: 50),
```

Pull out `50` into constant for readability?